.github/dependabot.yml

```diff
@@ -18,10 +18,30 @@ updates:
       interval: "daily"
     labels:
       - "kind/dependencies"
+
   - package-ecosystem: "npm"
     directory: "/website"
     schedule:
       interval: "daily"
     labels:
       - "kind/dependencies"
       - "area/website"
+
+  - package-ecosystem: "gomod"
+    directory: "/sdk/go"
+    schedule:
+      interval: "daily"
+    labels:
+      - "kind/dependencies"
+  - package-ecosystem: "npm"
+    directory: "/sdk/nodejs/dagger"
+    schedule:
+      interval: "daily"
+    labels:
+      - "kind/dependencies"
+  - package-ecosystem: "pip"
```

Comment:
/cc @helderco @samalba 

Is this alright? I see there's `pip`, `pipenv`, `pip-compile` and we're using `pyproject.toml` which I've never seen before

Reply:
`pyproject.toml` is used by `pip` so it should be good. It's simply a new alternative to `setup.py`/